_events/2020/2020-04-virtual-workshop.md

```diff
@@ -26,14 +26,15 @@ This workshop will be held in three online sessions:
     - Panel discussion (45 min)
 1. **Technical talks** - Short talks about projects that RSEs are working on (Wednesday 4/22, 3:00-4:30 PM EDT)
     - Presentations I (3x10 min)
-        - Fang (Cherry) Liu, Georgia Tech
-        - Junchao Xia, Princeton
-        - Pradeep Raamana, Baycrest Health Sciences
+        - TBA - Fang (Cherry) Liu, Georgia Tech
+        - Development of ASPIRE Python Package - Junchao Xia, Princeton
+        - Research data management for medical data with [pyradigm](https://github.com/raamana/pyradigm) - Pradeep Raamana, Baycrest Health Sciences
```

Comment:
Maybe we should be consistent with how talk titles are capitalized (or not)? The other argument I can see is if we are just representing what was sent, that would make sense I think too.

Reply:
I originally just cut and pasted what people sent me, retaining their capitalization.  But I can go through and make it consistent.

Reply:
I prefer consistency

Reply:
Will do